interactions/api/models/channel.py

```diff
@@ -724,10 +724,14 @@ async def purge(
 
         .. warning:: Calling this method can lead to rate-limits when purging higher amounts of messages.
 
+
```

Comment:
I think this new line is unnecessary when compared to the other docstrings with a code block.